scene/gui/container.cpp

```diff
@@ -178,7 +178,6 @@ Vector<int> Container::get_allowed_size_flags_vertical() const {
 void Container::_notification(int p_what) {
 	switch (p_what) {
 		case NOTIFICATION_ENTER_TREE: {
-			pending_sort = false;
```

Comment:
This shouldn't change anything except in cases when a pending sort was fired before exiting the tree, it will be false already at any other time, I don't think this is the right fix